src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/__snapshots__/index.test.jsx.snap

```diff
@@ -10,14 +10,39 @@ exports[`VideoSourceWidget snapshots snapshots: renders as expected with default
       className="border-primary-100 border-bottom pb-4"
     >
       <Form.Control
-        floatingLabel="Video ID or URL"
+        floatingLabel="Video ID"
         onBlur={[Function]}
         onChange={[MockFunction]}
         value=""
       />
+      <Component
+        className="text-primary-300 my-3"
+      >
+        <FormattedMessage
```

Comment:
<img width="971" alt="Screen Shot 2022-12-12 at 9 41 03 AM" src="https://user-images.githubusercontent.com/49422820/207073567-24e919dd-0a50-4d41-a122-6911fd94832e.png">
The bottom padding looks odd to me here, (in theory the label should be closer to the textinput it refers to?) But if it matches mocks all cool with that, I can't seem to find them.